packages/web-components/fast-foundation/docs/integrations/angular.md

```diff
@@ -87,6 +87,17 @@ export class AppComponent {
 }
 ```
 
+
+To allow NgModule to contain Non-Angular elements named, add  the code in your `app/app.module.ts` file contents with this:
```

Comment:
I think you could just say "Allows NgModule..." and there is a double space in ", add  the code".